src/pools/ssContracts/SideStaking.ts

```diff
@@ -21,11 +26,18 @@ export class SideStaking {
     this.config = config || configHelperNetworks[0]
   }
 
+  private sideStakingContract(ssAddress: string) {
+    return setContractDefaults(
+      new this.web3.eth.Contract(this.ssAbi, ssAddress),
+      this.config
+    )
+  }
+
   async amountToUnits(token: string, amount: string): Promise<string> {
```

Comment:
there are 3 places in the codebase with exactly the same code. Why? So if we do a change here do we need to change all other places too?

<img width="581" alt="Screen Shot 2022-04-14 at 16 45 13" src="https://user-images.githubusercontent.com/90316/163425997-7de18ab5-4f04-4a24-8221-50d2515e407d.png">

Reply:
not fully same code but this is just crazy confusing and there's no way for reviewer to know what the intention here was or why there are methods with exact same name in multiple files.

In any case this is then also populated to CodeClimate:

<img width="863" alt="Screen Shot 2022-04-14 at 16 48 06" src="https://user-images.githubusercontent.com/90316/163426462-62c907a0-6566-417e-86ef-5f06ae31baa2.png">